src/layouts/EditHomepage.jsx

```diff
@@ -954,192 +954,162 @@ const EditHomepage = ({ match }) => {
-                                      sectionIndex={sectionIndex}
-                                      highlights={
-                                        section.hero.key_highlights ?? []
-                                      }
-                                      onFieldChange={onFieldChange}
-                                      createHandler={createHandler}
-                                      deleteHandler={(event, type) => {
-                                        onOpen()
-                                        setItemPendingForDelete({
-                                          id: event.target.id,
-                                          type,
-                                        })
-                                      }}
-                                      shouldDisplay={
-                                        displaySections[sectionIndex]
-                                      }
-                                      displayHighlights={displayHighlights}
-                                      displayDropdownElems={
-                                        displayDropdownElems
-                                      }
-                                      displayHandler={displayHandler}
-                                      errors={errors}
-                                      handleHighlightDropdownToggle={
-                                        handleHighlightDropdownToggle
-                                      }
-                                    />
-                                  </Editable.EditableAccordionItem>
-                                  <Divider />
-                                  <VStack
-                                    spacing="0.5rem"
-                                    alignItems="flex-start"
-                                  >
-                                    <CustomiseSectionHeader />
-                                  </VStack>
-                                </>
-                              )}
-                              {section.resources && (
-                                <Editable.DraggableAccordionItem
-                                  draggableId={`resources-${sectionIndex}-draggable`}
-                                  index={sectionIndex}
-                                  tag={<Tag variant="subtle">Resources</Tag>}
-                                  title={section.resources.title}
-                                >
-                                  <EditorResourcesSection
+                        {frontMatter.sections.map((section, sectionIndex) => (
+                          <>
+                            {/* Hero section */}
+                            {section.hero && (
+                              <>
+                                <Editable.EditableAccordionItem title="Hero section">
+                                  <EditorHeroSection
                                     key={`section-${sectionIndex}`}
```

Comment:
could we uuid instead?

Reply:
not gonna do it here - this is going to be deleted in subsequent RPs